synfig-studio/images/CMakeLists.txt

```diff
@@ -269,7 +268,40 @@ foreach (IMAGE IN ITEMS ${IMAGES})
     )
 endforeach()
 
+# Splash Screen
+if(${CMAKE_BUILD_TYPE} MATCHES Debug)
+    # Get git details for splash_screen_development.sif.in
+    configure_file(
+        ${CMAKE_CURRENT_SOURCE_DIR}/splash_screen_development.sif.in
+        ${CMAKE_CURRENT_BINARY_DIR}/splash_screen_development.sif
+        @ONLY
+    )
+
+    add_custom_command(
```

Comment:
Why not put the image path ($ {CMAKE_CURRENT_BINARY_DIR} /splash_screen_development.sif) in the variable and move the `add_custom_command` out from the condition?